templates/repo/issue/view_content.tmpl

```diff
@@ -39,7 +39,13 @@
 						<div class="ui bottom attached segment">
 							<div class="ui small images">
 								{{range .Issue.Attachments}}
-									<a target="_blank" href="{{AppSubUrl}}/attachments/{{.UUID}}"><img class="ui image" src="{{AppSubUrl}}/attachments/{{.UUID}}"></a>
+									<a target="_blank" href="{{AppSubUrl}}/attachments/{{.UUID}}">
+										{{if FilenameIsImage .Name}}
+											<img class="ui image" src="{{AppSubUrl}}/attachments/{{.UUID}}" title='Click to see "{{.Name}}" in a new tab'>
```

Comment:
`title="..."` should use i18n right?
(same for the other 3 instances in this file in that case)